src/csharp/Grpc.Tools.Tests/ProtoCompileBasicTest.cs

```diff
@@ -30,6 +32,7 @@ public class ProtoCompileTestable : ProtoCompile
         {
             public string LastPathToTool { get; private set; }
             public string[] LastResponseFile { get; private set; }
+            public List<string> StdErrMessages = new List<string>();
```

Comment:
```suggestion
            public List<string> StdErrMessages { get; } = new List<string>();
```

Other public members are exposed as properties.